source/support/communicating-with-users/common-responses.html.md

```diff
@@ -306,7 +306,9 @@ Redirect: govuk-enquiries@digital.cabinet-office.gov.uk and/or contact the Polic
 
 ### File and format of new coat of arms
 
-> You can extract the image from GOV.UK Frontend v4.9.0 or v5.7.0. We provide the image as a SVG file. You can convert this to the format you need.
+> You can <a href="https://raw.githubusercontent.com/alphagov/govuk-frontend/9b57da6f0b8194473d67288666bc7d100339f289/packages/govuk-frontend/src/govuk/assets/images/govuk-crest.svg">download a SVG version of the coat of arms</a> from GOV.UK Frontend's repository. 
```

Comment:
The PR has been merged so the link can point to the main branch now (also Markdown-ified the link).

```suggestion
> You can [download a SVG version of the coat of arms](https://raw.githubusercontent.com/alphagov/govuk-frontend/refs/heads/main/packages/govuk-frontend/src/govuk/assets/images/govuk-crest.svg) from GOV.UK Frontend's repository. 
```

Reply:
Oh nice, thanks. I'd initialy put an `<a>` to use the `download` attribute before realising this would likely be sent by email/zendesk, likely breaking this feature.